### PR TITLE
[Cookbook] [Security] Use UserLoaderInterface instead of UserProviderInterface

### PR DESCRIPTION
**Pull request by ogizanagi**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.8+ |
| Fixed tickets | #5968 |

### CODE REVIEW

File: cookbook/security/entity_provider.rst [hunk 1]
```diff
@@ -458,30 +456,13 @@ interface requires three methods: ``loadUserByUsername($username)``,
...
-            return $this->getEntityName() === $class
-                || is_subclass_of($class, $this->getEntityName());
-        }
     }
 
-For more details on these methods, see :class:`Symfony\\Component\\Security\\Core\\User\\UserProviderInterface`.
+.. versionadded:: 2.8
+    The :class:`Symfony\\Bridge\\Doctrine\\Security\\User\\UserLoaderInterface`
+    interface was introduced in 2.8. Prior to Symfony 2.8, you had to implement
+    `Symfony\\Component\\Security\\Core\User\\UserProviderInterface` requiring you
```
Comment (WouterJ) [inline cookbook/security/entity_provider.rst]: you should use double backticks in reStructured Text and only a single `\`. I would also prefer to stop after the class name, removing "requiring you to implement three methods, where only one was useful.".

**Comment by WouterJ** [comment]
Cool! I didn't know this feature until now :)

I've left a minor comment, about syntax, after you've fixed that one, we can merge this imo (if you don't have time to fix it, just say it and we'll do it for you).

:christmas_tree:

**Comment by ogizanagi** [comment]
@WouterJ : Thanks for the review. PR updated !

**Comment by WouterJ** [comment]
:+1:

**Comment by xabbuh** [comment]
Thank you @ogizanagi.

**Pull request merged**